src/components/Panel.tsx

```diff
@@ -1,45 +1,75 @@
 import * as React from "react";
 import { GenericWedgeIcon } from "@nypl/dgx-svg-icons";
 
+// The Panel component comes with five built-in styles: "success" (green),
+// "warning" (yellow), "danger" (red), "instruction" (blue), and "default" (gray).
+// You can also define your own styles.  If you do not pass in a value for the style prop,
+// the style will be set to "default".
+
+// There are two ways to populate the panel with content: you can use either the body prop,
+// which accepts a JSX element, or the text prop, which accepts an HTML string.
+
+// To make a static (as opposed to a collapsible) Panel, set the collapsible prop to false.
+// This will: remove the open/close icon; prevent the panel from opening/closing when the header
+// is clicked; render the header as a div rather than a button; and automatically ensure that the
+// panel is always open (i.e. you do not need to also set the openByDefault prop).
+
```

Comment:
We don't have a standard to comment classes but I suggest we use esdoc https://esdoc.org/manual/usage.html. We've used it in other projects and they are helpful, if not for the rendered documentation then for the standard in writing comments. This will really benefit what you wrote about into better organized comments.

Reply:
In the next round of changes; going to commit the others first and then go back to this!

Reply:
Don't forget...

Reply:
After two hours of (unsuccessfully) messing around with esdoc: this is too big a task to go into this branch.  I'm also not convinced that esdoc is the best thing to use; its standard for writing comments doesn't seem like the right format for this kind of comment, and getting it to work with React and Typescript requires a bunch of separate plug-ins, not all of which seem totally stable.  (Out of curiosity: why do you prefer it over, say, [React Styleguidist](https://react-styleguidist.js.org/)?)  Also, it might be better to put these types of comments into the Storybook documentation and have everything in one place, rather than having two separate sets of documentation (I know they document very different types of thing, but still).  But anyway, this is a conversation for a different PR!

Reply:
Yes, they are different things. We've used React Styleguidist before but Storybook has more support (it's also not exactly comparable to esdoc). I just wanted the comment documentation style, not any plugins that go with it but it's good to try. I think Storybook can read esdoc or jsdoc strings so another reason I wanted to use it. Yes, next PR.